src/components/[guild]/RoleCard/components/EditRewardDescription/EditRewardDescriptionModal.tsx

```diff
@@ -0,0 +1,22 @@
+import { ModalContent, ModalOverlay, ModalProps } from "@chakra-ui/react"
+import { Modal } from "../../../../common/Modal"
+import React from "react"
+import { GuildPlatform } from "../../../../../types"
+import { EditRewardModalContent } from "./EditRewardModalContent"
+
+type Props = {
+  guildPlatform: GuildPlatform
+} & Omit<ModalProps, "children">
+
+export const EditRewardDescriptionModal: React.FC<Props> = ({
+  guildPlatform,
+  isOpen,
+  onClose,
+}: Props): JSX.Element => (
+  <Modal isOpen={isOpen} onClose={onClose} size={"xl"}>
+    <ModalOverlay />
+    <ModalContent>
+      <EditRewardModalContent guildPlatform={guildPlatform} onClose={onClose} />
```

Comment:
Why is it separated into another component? They can't be used separately and the files aren't too big, I think we should keep this logic in one file

Reply:
I thought it is good if I don't load the forms logic just after the modal is opened. Like this it doesn't need to reset it.

Reply:
I see, makes sense! In this case, we should keep the two components, but move the other one in this file too, so the logic stays batched. + would be nice to add a comment about it